java/rocksjni/rocksjni.cc

```diff
@@ -688,8 +693,8 @@ jint rocksdb_get_helper(
     return kStatusError;
   }
 
-  int cvalue_len = static_cast<int>(cvalue.size());
-  int length = std::min(jentry_value_len, cvalue_len);
+  jint cvalue_len = static_cast<int>(cvalue.size());
```

Comment:
You are right, I forgot to update template argument. Important here was the final type. jint is int, but std:min failed on type conversion. I can update parameter to jint.